docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+
+
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+GET http://localhost:1337/restaurants?_locale=fr
+```
+
+:::
+
+::: tab Response
+
+**Example response**
+
+```json
+[
+  {
+    "id": 4,
+    "name": "Restaurant A",
+    "slug": "restaurant-a-1",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 0,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant gastronomique dans la vallée.",
+    "menu": null,
+    "locale": "fr",
+    "published_at": "2021-04-07T10:12:50.247Z",
+    "created_at": "2021-04-07T10:12:33.324Z",
+    "updated_at": "2021-04-07T10:12:50.315Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": [
+      {
+        "id": 3,
+        "locale": "en",
+        "published_at": "2021-04-07T10:10:31.949Z"
+      }
+    ]
+  },
+  {
+    "id": 8,
+    "name": "Restaurant F",
+    "slug": "restaurant-f",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant face à la mer",
+    "menu": null,
+    "locale": "fr",
+    "published_at": "2021-04-07T13:22:46.589Z",
+    "created_at": "2021-04-07T13:22:46.617Z",
+    "updated_at": "2021-04-07T13:22:46.617Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+  }
+]
+```
+
+In the example response above:
+
+- restaurant with `"id": 4` is a French (`"locale": "fr"`) localization of the existing restaurant with `"id": 3` (created for the default `en` locale), as shown in the `localizations` object included in the response (see [creating a localization for an existing entry](#creating-a-localization-for-an-existing-entry)).
+- restaurant with `"id": "8"` was created from scratch using the API, passing the `locale: fr` in the request body (see [creating a new localized entry](#creating-a-new-localized-entry)).
+:::
+::::
+
+### Creating a new localized entry
+
+To create a localized entry from scratch, send a POST request to the Content API.
+
+If no locale has been passed in the request body, the entry is created using the default locale for the application:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant B",
+  "slug": "restaurant-b",
+  "stars": 1,
+  "description": "Restaurant with terrace in front of the lake"
+}
+```
+
+:::
+
+::: tab Response
+
+**Example response**
```

Comment:
Same as the previous comment for the simplification of the JSON object and use some FoodAdvisor data.